packages/sanity/src/core/form/inputs/ReferenceInput/PreviewReferenceValue.tsx

```diff
@@ -81,3 +153,45 @@ export function PreviewReferenceValue(props: {
     />
   )
 }
+
+function UnavailableMessage(props: {icon: ComponentType; children: ReactNode; title: ReactNode}) {
+  const Icon = props.icon
+  return (
+    <Flex padding={3}>
+      <Box>
+        <Text size={1}>
+          <Icon />
+        </Text>
+      </Box>
+      <Box flex={1} marginLeft={3}>
+        <Text size={1} weight="semibold">
+          {props.title}
+        </Text>
+
+        <Box marginTop={3}>
+          <Text as="p" muted size={1}>
+            {props.children}
+          </Text>
+        </Box>
+      </Box>
+    </Flex>
+  )
+}
+
+const humanizeList = (list: React.ReactNode[], conjunction: string) => {
+  if (list.length === 1) {
+    return list[0]
+  }
+
+  if (list.length === 2) {
+    return [list[0], <Fragment key="comma"> {conjunction} </Fragment>, list[1]]
+  }
+
+  const subList = list.slice(0, -1)
+  return [
+    ...subList.map((item, i) => <Fragment key={i}>{item}, </Fragment>),
+    <Fragment key="last">
+      {conjunction} {list.at(-1)}
```

Comment:
We still use ES2017 as our build target, but `Array.prototype.at` is not included until ES2022.
```suggestion
      {conjunction} {list[list.length - 1]}
```